### PR TITLE
Adding jupytab

### PR DESCRIPTION
**Pull request by js-dieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by js-dieu** [comment]
Ok, for being maintainer

File: recipes/jupytab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "jupytab" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/jupytab/meta.yaml:9]: The build is failing as the sdist on PyPI is broken. The `VERSION` file isn't included:

```bash
$ curl --silent -LO https://pypi.io/packages/source/j/jupytab/jupytab-0.9.7.tar.gz
$ tar xvf jupytab-0.9.7.tar.gz
jupytab-0.9.7/
jupytab-0.9.7/jupytab.egg-info/
jupytab-0.9.7/jupytab.egg-info/dependency_links.txt
jupytab-0.9.7/jupytab.egg-info/PKG-INFO
jupytab-0.9.7/jupytab.egg-info/top_level.txt
jupytab-0.9.7/jupytab.egg-info/requires.txt
jupytab-0.9.7/jupytab.egg-info/SOURCES.txt
jupytab-0.9.7/MANIFEST.in
jupytab-0.9.7/tests/
jupytab-0.9.7/tests/test_dataframe.py
jupytab-0.9.7/tests/__init__.py
jupytab-0.9.7/tests/test_function.py
jupytab-0.9.7/tests/test_util.py
jupytab-0.9.7/PKG-INFO
jupytab-0.9.7/README.md
jupytab-0.9.7/setup.py
jupytab-0.9.7/jupytab/
jupytab-0.9.7/jupytab/table.py
jupytab-0.9.7/jupytab/function.py
jupytab-0.9.7/jupytab/dataframe_table.py
jupytab-0.9.7/jupytab/__init__.py
jupytab-0.9.7/setup.cfg
```

As it looks like your a developer of the package, the fix is to add this filename to a `MANIFEST.in` file.

Alternatively I'd recommend considering `setuptools_scm` as it does a great job of using the git metadata to manage the version automatically.

Comment (js-dieu) [reply recipes/jupytab/meta.yaml:9]: Hi @chrisburr 

Thanks for having a look at it. I apologize, I should have mentioned that I did fix that point on Friday through appropriate PR (which is merged) and was waiting for jupytab’s maintainer to make proper delivery on Pypi. 
Now that’s done, I will finish the job this monday morning.

**Comment by js-dieu** [comment]
Still waiting for delivery from jupytab. Closing PR.

**Pull request closed without merging**